hls4ml/backends/fpga/fpga_backend.py

```diff
@@ -860,6 +860,64 @@ def generate_conv2d_line_buffer_fn(
 
         return generated_code
 
+    def generate_pointwise_conv1d_fn(self, layer_idx, reuse_factor=1):
```

Comment:
If we're placing this here, it's available for all backends, including quartus, but the rest of the changes doesn't seem to touch quartus?

Reply:
Moved to `vivado/vivado_backend.py`